src/pages/home/report/ReportActionCompose.js

```diff
@@ -208,6 +215,10 @@ class ReportActionCompose extends React.Component {
                                                 onPress={(e) => {
                                                     e.preventDefault();
                                                     this.setMenuVisibility(true);
+
+                                                    /* Keep last focus inside the input so that focus is restored
+                                                     on modal close. Otherwise breaks modal 2 modal transition */
+                                                    this.focus();
```

Comment:
@kidroca I know this was a while ago, but could you potentially help me out with some reproduction steps for the issue you described here? i.e: If I remove this code how can I reproduce the modal 2 modal transition issue?

Reply:
@roryabraham 
You won't be able to. This is a leftover that should be removed. It was brought up here: https://github.com/Expensify/Expensify.cash/pull/2022#issuecomment-820330081

---

After this line is removed only on iOS, and only when the input field is initially focused there would be a small quirk in the modal to modal transition, but it was always there (even with the setTimeout fix). It might not be an issue in prod. 
Check my comments and videos. There's a full explanation and a possible fix.

Reply:
Okay, so in your opinion is this safe to remove for now? (maybe with a follow-up to resolve some minor issues)?